docs/design/protip.rst

```diff
@@ -0,0 +1,37 @@
+ProTip template to enhancing user experience
```

Comment:
If ProTip is going to come into our vocabulary, could you please add it to the .github/styles/config/vocabularies/Mautic/accept.txt file so it's recognised as a valid term? Thanks!

Reply:
Done!